MovieApp/Scenes/Cell/CategoryCell.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  CategoryCell.swift
+//  MovieApp
+//
+//  Created by Ana Dzebniauri on 18.07.23.
+//
+
+import UIKit
+
+final class CategoryCell: UICollectionViewCell {
+    
+    //MARK: - Properties
+    private lazy var categoryLabel: UILabel = {
+        let categoryLabel = UILabel()
+        categoryLabel.clipsToBounds = true
+        categoryLabel.text = Constants.CategoryLabel.text
+        categoryLabel.textColor = .white
+        categoryLabel.font = Constants.Font.medium
+        categoryLabel.translatesAutoresizingMaskIntoConstraints = false
+        return categoryLabel
+    }()
```

Comment:
შიდა კონტექსტში, კომპონენტის შექმნისას სადაც ცხადია რომ categoryLabel-ზეა საუბარი, მის ნაცვლად შეგიძლია უფრო მოკლე სახელი, მაგალითად `label` გამოიყენო
```suggestion
    private lazy var categoryLabel: UILabel = {
        let categoryLabel = UILabel()
        categoryLabel.clipsToBounds = true
        categoryLabel.text = Constants.CategoryLabel.text
        categoryLabel.textColor = .white
        categoryLabel.font = Constants.Font.medium
        categoryLabel.translatesAutoresizingMaskIntoConstraints = false
        return categoryLabel
    }()
```